server/db/models/user.js

```diff
@@ -5,10 +5,12 @@ var _ = require('lodash');
 
 var schema = new mongoose.Schema({
     email: {
-        type: String
+        type: String,
+        required: true
```

Comment:
FSG doesn't set up their email and password fields as required, because users may decide to use facebook/twitter/google to log in.

Something to think about changing if/when we add social log in